var/spack/repos/builtin/packages/py-pyonmttok/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyonmttok(CMakePackage, PythonExtension):
+    """Tokenizer is a fast, generic, and customizable text tokenization library for
+    C++ and Python with minimal dependencies."""
+
+    homepage = "https://github.com/OpenNMT/Tokenizer/"
+    git = "https://github.com/OpenNMT/Tokenizer.git"
+
+    maintainers("meyersbs", "aweits")
+
+    version("1.37.1", tag="v1.37.1", submodules=True)
+
+    extends("python")
+
+    # Uses setuptools.installer, deprecated in:
+    # https://setuptools.pypa.io/en/stable/history.html#v59-1-0
```

Comment:
Deprecated or removed? If it still works in newer versions, it might make things easier to concretize by supporting them for now.

Reply:
I would remove the upper bound if newer versions still work

Reply:
I tested `spack install py-pyonmttok ^py-setuptools@59.4.0` and `^py-setuptools@62.3.2` and it installs, but I get the following error on import:
```
>>> import pyonmttok
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bsmits/spack/opt/spack/linux-rhel7-skylake_avx512/gcc-11.2.0/py-pyonmttok-1.37.1-wygtgxzeop4bqwerxgvnnrlfj5kge4tx/lib/python3.8/site-packages/pyonmttok/__init__.py", line 20, i
n <module>
    from pyonmttok._ext import (
ModuleNotFoundError: No module named 'pyonmttok._ext'
```

Reply:
And this error occurs for both setuptools or only newer setuptools?

Reply:
Looks like I'm suddenly getting that for all versions of setuptools that I've tested. @qwertos can you poke?